src/docs/devices/KinCony-A32Pro/index.md

```diff
@@ -0,0 +1,1071 @@
+---
+title: KinCony-A32 Pro (ESP32-S3 32CH Relay Controller)
+date-published: 2024-10-12
+
+type: relay
+standard: global
+board: esp32
+---
+
+![Product](A32Pro-10.jpg "Product Image")
+![Product](A32Pro_diagram.jpg "Product Image")
+
+## Resources
+
+- [ESP32 pin define details](https://www.kincony.com/forum/showthread.php?tid=5519)
+- [YouTube video tour](https://youtu.be/yqArOKF5vWc)
+
+## ESPHome Configuration
+
+Here is an example YAML configuration for the KinCony-A32 Pro board.
+
+```yaml
+esphome:
+  name: a32-pro-arduino
+  friendly_name: a32-Pro-arduino
+  platformio_options:
+    board_build.extra_flags:
+      # WIFI_CONTROL_SELF_MODE = 0
+      # WIFI_CONTROL_SELF_MODE = 1
+      - "-DWIFI_CONTROL_SELF_MODE=0"
+
+external_components:
+  - source:
+      type: git
+      url: https://github.com/hzkincony/esphome-tuya-wifi-mcu
+      ref: v1.2.0
+
+esp32:
+  board: esp32-s3-devkitc-1
+  framework:
+    type: arduino
+
+# Enable logging
+# logger:
+
+# Enable Home Assistant API
+api:
+  encryption:
+    key: "fBpihEYPVlfKaE4ZIcY0uRcJ134u7DBKzzskk994+Lw="

```

Comment:
Please remove this value from the file. There should be no passwords or secrets references in any examples. The only exceptions are !secret wifi_ssid and !secret wifi_password.